fala/templates/adviser/accessibility_statement.html

```diff
@@ -0,0 +1,117 @@
+
+    <h1 class="govuk-heading-xl">Accessibility statement for Find a legal advisor</h1>
+
+    <p class="govuk-body">
+      This accessibility statement applies to the ‘Find a legal advisor’ service.
+    </p>
+
+    <p class="govuk-body">
+      This website is run by the <a class="govuk-link" href="https://www.gov.uk/government/organisations/legal-aid-agency" target="_blank" rel="noopener external noreferrer">Legal Aid Agency</a> (<abbr>LAA</abbr>), part of the Ministry of Justice (<abbr>MoJ</abbr>).
+      We want as many people as possible to be able to use this website. For example, that means you should be able to:
+    </p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>change colours, contrast levels and fonts using browser or device settings</li>
+      <li>zoom in up to 400% without the text spilling off the screen</li>
+      <li>navigate most of the website using a keyboard or speech recognition software</li>
+      <li>listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)</li>
+    </ul>
+
+    <p class="govuk-body">
+      We’ve also made the website text as simple as possible to understand.
+    </p>
+
+    <p class="govuk-body">
+      <a class="govuk-link" href="https://mcmw.abilitynet.org.uk/" target="_blank" rel="noopener external noreferrer">AbilityNet</a> has
+      advice on making your device easier to use if you have a disability.
+    </p>
+
+    <h2 class="govuk-heading-m">How accessible this website is</h2>
+
+    <p class="govuk-body">
+      We know some parts of this website are not fully accessible:
+    </p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>elements of the results page do not meet minimum contrast standards</li>
+      <li>Google Translate does not translate technical terms into Plain English</li>
+      <li>hyperlinks to google maps are not specific about the listing they are related to</li>
+    </ul>
+   
+    <h2 class="govuk-heading-m">Feedback and contact information</h2>
+
+    <p class="govuk-body">
+      We’re always looking to improve the accessibility of this website. 
+      If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact us by email at 
+      <a class="govuk-link email" href="mailto:eligibility@justice.gov.uk">eligibility@justice.gov.uk</a>.
+    </p>
+
+    <h2 class="govuk-heading-m">Enforcement procedure</h2>
+
+    <p class="govuk-body">
+      The Equality and Human Rights Commission (<abbr>EHRC</abbr>) is responsible for enforcing the Public
+      Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations
+      2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to
+      your complaint, <a class="govuk-link" href="https://www.equalityadvisoryservice.com/" target="_blank" rel="noopener external noreferrer">contact the Equality
```

Comment:
Checked with content designer: the links within the accessibility statement and privacy policy should not open in a new tab.